installer/mac/symphony-mac-packager.pkgproj

```diff
@@ -715,9 +704,9 @@
 						<key>VALUE</key>
 						<dict>
 							<key>PATH</key>
-							<string>intro.html</string>
+							<string>/Users/vishwas/Code/symphony/SymphonyElectron/installer/mac/intro.html</string>
```

Comment:
This looks like it might be unintentional - path with user name?

Reply:
Wow! Great catch @mattias-symphony. I'll fix it and update the master branch!